server/libs/__tests__/dateHelpers.spec.js

```diff
@@ -0,0 +1,31 @@
+const { addDays } = require('../dateHelpers');
+
+describe('Date helper', () => {
+    it('should add a day and be 29 Feb on leap year', () => {
+        const date = addDays('2020-02-28', 1);
+
+        expect(date).toBeDefined;
+        expect(date).toEqual(new Date('2020-02-29'));
+    });
+
+    it('should add a day normally not on leap year', () => {
+        const date = addDays('2021-02-28', 1);
+
+        expect(date).toBeDefined;
+        expect(date).toEqual(new Date('2021-03-01'));
+    });
+
+    it('should add a day normally', () => {
+        const date = addDays('2021-01-01', 1);
+
+        expect(date).toBeDefined;
+        expect(date).toEqual(new Date('2021-01-02'));
+    });
+
+    it('should roll a year over on adding day', () => {
+        const date = addDays('2020-12-31', 1);
+
+        expect(date).toBeDefined;
+        expect(date).toEqual(new Date('2021-01-01'));
+    });
+});
```

Comment:
This is less important, but ideally there should be line endings at the end of files so that it plays nicer with `git diff` in future (as to add a line 32 you'd have to edit line 31 to stick a line ending on the end).

Reply:
Rectified 👍🏼